python_src/src/lamp_py/tableau/jobs/gtfs_rail.py

```diff
@@ -0,0 +1,400 @@
+import os
+
+import pyarrow
+import pyarrow.parquet as pq
+import sqlalchemy as sa
+
+from lamp_py.tableau.hyper import HyperJob
+
+
+class HyperGTFS(HyperJob):
+    """Base Class for GTFS Hyper Jobs"""
+
+    def __init__(
+        self,
+        gtfs_table_name: str,
+        table_query: str,
```

Comment:
`table_query` should constructed as SELECT statement  with a `%s` placeholder for WHERE statement related  to `static_version_key`